quickstart/src/main/AndroidManifest.xml

```diff
@@ -1,19 +1,19 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="com.twilio.video.quickstart">
```

Comment:
This formatting is applied by Android Studio when adding an activity so I guess this is correct